tests/io/vasp/test_inputs.py

```diff
@@ -555,11 +555,12 @@ def test_copy(self):
         assert self.incar.get("LDAU") is None
 
     def test_diff(self):
-        filepath1 = f"{TEST_FILES_DIR}/INCAR"
+        filepath1 = f"{VASP_IN_DIR}/INCAR"
         incar1 = Incar.from_file(filepath1)
-        filepath2 = f"{TEST_FILES_DIR}/INCAR.2"
+        filepath2 = f"{VASP_IN_DIR}/INCAR_2"
+        filepath3 = f"{VASP_IN_DIR}/INCAR_3"
         incar2 = Incar.from_file(filepath2)
-        incar3 = Incar.from_file(filepath2)
+        incar3 = Incar.from_file(filepath3)
```

Comment:
Need to look closer into this part. The origin test seems to be pointing to a complete wrong test `INCAR` file: 
```python
incar2 = Incar.from_file(filepath2)
incar3 = Incar.from_file(filepath2)
```
But tests are passing.

Even after correcting the file path, the this test is still passing? Need to take a closer look soon.

Reply:
It appears `INCAR_3` and `INCAR_2` are identical. `INCAR_3` has been removed